pages/sections/landing.tsx

```diff
@@ -34,8 +34,8 @@ export default function Landing() {
 
   return (
     <>
-      <section className="relative flex flex-col-reverse w-full h-screen md:min-h-[65rem] my-0 md:mb-20 gap-6">
-        <div className="absolute left-1/2 -translate-x-1/2 w-full top-[30%] md:top-[25%] lg:top-[16%] flex flex-col justify-center items-center z-10 px-4">
+      <section className="relative flex flex-col-reverse w-full h-screen md:min-h-[60rem] mt-4 gap-6">
```

Comment:
for the margins, im not sure if it looked different on ur laptop but on my the new changes look too cramped?

this is how it looks with ur code
![image](https://user-images.githubusercontent.com/62922309/221084515-cc5567b8-207e-42b9-a5c4-3347d1348c6a.png)

and this is how it used to look with the old code
![image](https://user-images.githubusercontent.com/62922309/221084617-6cfd0851-f543-4da6-ba83-a63cc3d0b73a.png)

I personally think the old margins look better but what do you think?

Reply:
lol star pupil mika told me to make those changes, ill revert them

Reply:
pls ok the margins were gonna get fixed in my PR ✋🏻

Reply:
Oo yea that woulda been fine too but looks like the director alr fixed it 🙌